core/src/main/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoEntityProviderBuilder.scala

```diff
@@ -78,6 +78,8 @@ class DataRepoEntityProviderBuilder(workspaceManagerDAO: WorkspaceManagerDAO, da
     // trigger any exceptions
     val dataRef = dataRefTry.get
```

Comment:
Maybe not part of this PR, but we currently differentiate the response between when a reference does not exist in a workspace and when the user does not have access to the reference (see lines 72). Is this behavior okay or do we want to obscure for security reasons?

Reply:
I think this is ok in this case. Any user who reaches this point has read access on the workspace, which means they can list the references that exist in this workspace. So they already have the ability to know which references do/do not exist; we're not leaking info.